docs/introduction.md

```diff
@@ -0,0 +1,59 @@
+# Introduction to Personalization
+
+Website Personalization is the process of creating customized experiences for visitors to a website. Rather than providing a single experience for all visitors, website personalization allows companies to present visitors with unique experiences tailored to their needs and desires.
+
+## What does this SDK offer?
+
+This SDK offers both Geo and Interest targeting. Below we will go into further detail for each type of targeting.
+
+## Geotargeting Functionality
+
+Geotargeting is the method of delivering different content to visitors based on their geolocation. This includes country, region, city, and other criteria.
+
+### Default Behavior and Available Customizations
+
+Using the `get_geo` function, engineers can get location specific data from site visitors, including `country`, `region`, `city`, `continent`, `conn-speed`, `conn-type`, `lat`, `lon`, and `latlon`. Engineers can modify these data types using the `pantheon.ei.geo_allowed_values` filter.
```

Comment:
We're currently not getting latitude/longitude from the CDN, so these should be omitted from the docs even though the code supports those parameters being returned.

```suggestion
Using the `get_geo` function, engineers can get location specific data from site visitors, including `country`, `region`, `city`, `continent`, `conn-speed`  (connection speed, e.g. `broadband`) and `conn-type` (connection type, e.g. `wireless`). Engineers can modify these data types using the `pantheon.ei.geo_allowed_values` filter.
```